canonical-web-and-design/ubuntu.com
# Redirect /aws/pro to /support

## Done

Redirect /aws/pro to /support

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/aws/pro
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that /aws/pro redirects to /support
